templates/accounts/donate.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+<div id="content_full"> <!-- START of Content area -->
+
+  <h1>Donations</h1>
+  <p>
+  Thank you for supporting Freesound, you can complete the following form with the name that will appear with the donation:
+  </p>
+  <!-- donations -->
+    <div id="donation_form">
+      {% if debug %}
+        <form name="_xclick" action="https://www.sandbox.paypal.com/yt/cgi-bin/webscr" method="post">
+      {% else %}
+        <form name="_xclick" action="https://www.paypal.com/yt/cgi-bin/webscr" method="post">
+      {% endif %}
+          <input type="hidden" name="cmd" value="_donations">
+          <input type="hidden" name="currency_code" value="EUR">
+          <input type="hidden" name="business" value="{{ paypal_email }}">
+          <input type="hidden" name="item_name" value="Freesound">
+          <label>Name</label><input type="text" name="custom" value="Anonymous">
+          <input type="hidden" name="notify_url" value="{{ request.scheme }}://{{ request.get_host }}{% url 'donation-complete' %}">
```

Comment:
We we do: https://github.com/MTG/freesound/blob/4e4b91fa45adabb5294487eeed384596425fd8bb/general/templatetags/absurl.py